lib/vdc/BOVCollection.cpp

```diff
@@ -7,12 +8,10 @@
 #include <fstream>
 #include <type_traits>
 #include "vapor/VAssert.h"
+#include "vapor/utils.h"
+#include "vapor/FileUtils.h"
 #include <stdio.h>
```

Comment:
Why don't you also switch this header to its C++ counterpart: cstdio?

Reply:
Thanks, fixed.  Pushing soon.